charts/authentik/ci/ct-extracontainer-values.yaml

```diff
@@ -0,0 +1,8 @@
+worker:
```

Comment:
Pretty sure you need the boilerplate that is in ct-values.yaml for this to work.

Reply:
This is no boilerplait, that is a "chart-testing lint (unit)test"

it is not needed, but if you want to pretend such bug it is helpfull

Reply:
I meant that you need some basic values apart from what you are testing for it to work, like what is `ct-values.yaml` and in all other tests we currently have.